src/index.html

```diff
@@ -5,11 +5,28 @@
     <title>SVGOMG - SVGO's Missing GUI</title>
     <meta name="theme-color" content="#303f9f">
     <meta name="viewport" content="width=device-width, minimum-scale=1.0">
-    <link rel="preload" as="script" href="js/page.js">
-    <link rel="preload" as="script" href="https://www.google-analytics.com/analytics.js">
-    <link rel="preload" as="style" href="all.css">
-    <link rel="preload" as="style" href="https://fonts.googleapis.com/css?family=Roboto:400,700%7CInconsolata">
     <style>{{headCSS|safe}}</style>
+
+    <script>
+      // Using preloads then JS to avoid keeping the browser spinner spinning.
+      function appendScript(url) {
+        var script = document.createElement('script');
+        script.src = url;
+        document.head.appendChild(script);
+      }
+      function appendStyle(url) {
+        var link = document.createElement('link');
+        link.rel = 'stylesheet';
+        // So it can be cached without no-cors in the service worker:
+        link.crossOrigin = true;
+        link.href = url;
+        document.head.appendChild(link);
+      }
+    </script>
+    <link rel="preload" crossorigin as="style" href="all.css" onload="appendStyle(href)">
+    <link rel="preload" crossorigin as="style" href="https://fonts.googleapis.com/css?family=Roboto:400,700%7CInconsolata" onload="appendStyle(href)">
+    <link rel="preconnect" crossorigin href="https://fonts.googleapis.com">
```

Comment:
Why preconnect to a domain used one line above? That should have no effect, right?

Are you trying to preconnect to the domain Google Fonts uses for the actual font files? aka https://fonts.gstatic.com

Reply:
Yes, I am. Good catch!